src/ripple/app/ledger/Ledger.h

```diff
@@ -420,6 +416,13 @@ class Ledger
         return mHash;
     }
 
+    // Updates the fees cached in the ledger.
+    // Safe to call concurrently. We shouldn't be storing
+    // fees in the Ledger object, they should be a local side-structure
+    // associated with a particular module (rpc, tx processing, consensus)
+    //
+    void deprecatedUpdateCachedFees() const;
```

Comment:
The whole point of this refactor was to make the function public, are you suggesting I undo the change?